site/layouts/shortcodes/ghrelease.html

```diff
@@ -0,0 +1 @@
+<a href="{{$.Page.Site.Params.github_repo}}/releases/download/v{{$.Page.Site.Params.release_version}}/{{ .Get 0 }}-{{$.Page.Site.Params.release_version}}.zip" data-proofer-ignore >{{ .Get 0 }}-{{$.Page.Site.Params.release_version}}.zip</a>
```

Comment:
Should this always be `data-proofer-ignore` ? I would think this should be an optional attribute, like ghlink has.

We will want this link tested long term.

Reply:
That's correct. I thought of the situation when we don't created release archives, but have updated the version. I will update this PR.